src/components/Pickups/PickupItem.vue

```diff
@@ -10,7 +10,7 @@
           <slot>Date or Store Slot</slot>
         </div>
         <div
-          class="description multiline"
+          class="description"
```

Comment:
Why did you change this? The pickup comment is supposed to support line breaks, as the input field is a textarea.

Reply:
Multiline adds a linebreak at the very beginning, making it look like this:
![multiline-on](https://user-images.githubusercontent.com/16825880/34676717-1254870a-f48e-11e7-929b-a531c54e475a.png)

Without it, it still supports line breaks, and looks like this:
![multiline-off](https://user-images.githubusercontent.com/16825880/34676724-19901976-f48e-11e7-8ff4-ef33be0ed9d9.png)

Reply:
I mean intentional linebreaks, when somebody hits 'enter' to create a newline.

Reply:
I entered some linebreaks. It should show like this:

![image](https://user-images.githubusercontent.com/4410802/34685700-88ac56ce-f4a9-11e7-970c-2a0581e97906.png)

With this PR, it shows like this:

![image](https://user-images.githubusercontent.com/4410802/34685741-ad4f0094-f4a9-11e7-9bb6-39546a6d5f4f.png)

Reply:
The preceding space is annoying though, I wonder where it comes from...

Reply:
Haha, I found it! :laughing: it was in the HTML - see my last commit